src/form.ts

```diff
@@ -206,11 +206,18 @@ export class Field<R, V> {
     throw new Error("This is a function to enable type introspection");
   }
 
-  getRequiredError(context: any): string {
-    if (typeof this.requiredError === "string") {
-      return this.requiredError;
+  requiredErrorType(context: any, requiredError: string | ErrorFunc): string {
```

Comment:
This needs a better name and it's not a method; it's a function. 'getRequiredError(context, requiredError)' is okay as a top-level function.